src/zepben/evolve/model/busbranch/__init__.py

```diff
@@ -0,0 +1,7 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
```

Comment:
idk what's happening but i thought the idea codestyles was configured to use the current year, do you wanna check/change these to 2021?